<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qCoreParametersWidget</class>
 <widget class="QWidget" name="qCoreParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>399</width>
        <height>731</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>2</number>
       </property>
       <item row="7" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>321</width>
           <height>181</height>
          </size>
         </property>
         <property name="title">
          <string>NeumannSet</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="margin">
           <number>0</number>
          </property>
          <item row="1" column="1" colspan="3">
           <widget class="QComboBox" name="NeumannSetSide">
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Top</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bot</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Side</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SideAll</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="NeumannSetId"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Id</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Equation</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QTableWidget" name="NeumannSetTable">
            <column>
             <property name="text">
              <string>Side</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>ID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Equation</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="NeumannSetEquation"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Side</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="4">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="NewmannSetAdd">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="NewmannSetDel">
              <property name="text">
               <string>Delete Row</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="Geometry">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Volume</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Surface</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>321</width>
           <height>101</height>
          </size>
         </property>
         <property name="title">
          <string>Extrude</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="horizontalSpacing">
           <number>-1</number>
          </property>
          <property name="verticalSpacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="ExtrudeHeight"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Divisions</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ExtrudeDivisions"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="MeshInfo">
         <property name="text">
          <string>Mesh Info</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Save Parallel</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="UnknownsVars"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>User Defined</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Output Mesh File</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Merge Tolerance</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Problem Type</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Geometry</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="ProblemType">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Geometry</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mesh</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="Info">
         <property name="text">
          <string>Info</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="SaveParallel">
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Multiple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Both</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>One</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="MergeTolerance"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="OutputFile"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ProblemType</tabstop>
  <tabstop>Geometry</tabstop>
  <tabstop>MergeTolerance</tabstop>
  <tabstop>OutputFile</tabstop>
  <tabstop>SaveParallel</tabstop>
  <tabstop>Info</tabstop>
  <tabstop>MeshInfo</tabstop>
  <tabstop>NeumannSetTable</tabstop>
  <tabstop>NewmannSetDel</tabstop>
  <tabstop>NewmannSetAdd</tabstop>
  <tabstop>ExtrudeHeight</tabstop>
  <tabstop>ExtrudeDivisions</tabstop>
  <tabstop>UnknownsVars</tabstop>
  <tabstop>scrollArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
